.travis.yml

```diff
@@ -11,3 +11,7 @@ branches:
   only:
     - master
     - 1.x_dev
+
+script:
+  - xcodebuild -workspace push-sdk.xcworkspace -scheme push-sdk -sdk iphonesimulator build
+  - xcodebuild -workspace push-sdk.xcworkspace -scheme push-sdk -destination 'platform=iOS Simulator,name=iPhone 7,OS=10.0' test
```

Comment:
@danielpassos @jcesarmobile cheers, was just curious, so as to try get a little familiarity around the iOS stuff :)